vizro-core/tests/unit/vizro/models/_components/form/test_range_slider.py

```diff
@@ -226,20 +226,6 @@ def test_validate_step_invalid(self):
         ):
             vm.RangeSlider(min=0, max=10, step=11)
 
-    @pytest.mark.parametrize(
```

Comment:
Instead of deleting these tests, shall we just re-write them?

Or is there any other unit test that shows what happens when you define steps / marks and all cross-combinations of these?